src/work-allocation-2/containers/task-assignment/task-assignment-container.component.scss

```diff
@@ -0,0 +1,6 @@
+.task-assignment-btn { 
```

Comment:
This styling does not affect anything and should not be added again - File should be removed from PR